charts/tidb-operator/values.yaml

```diff
@@ -217,14 +217,17 @@ admissionWebhook:
     ## apiservice config
     ## refer to https://kubernetes.io/docs/tasks/access-kubernetes-api/configure-aggregation-layer/#contacting-the-extension-apiserver
     insecureSkipTLSVerify: true
-    ## The key and cert for `tidb-admission-webook.<Release Namespace>` Service.
+    ## The tls key and cert for `tidb-admission-webook.<Release Namespace>.svc` Service in Secret.
```

Comment:
The `cert` and `key` fields are removed, is this description still necessary?

Reply:
The note is updated.